### DISCUSSION TITLE
after login, verify code page should show as modal, but width-full now

### DISCUSSION THREAD

**Discussion by wuyidebh** [backstagephp/filament-2fa#71] [Q&A]
after login, verify code page should show as modal, but width-full now
![dd](https://github.com/user-attachments/assets/c5f930a0-4c62-4b25-939d-e49adbadd450)

**Comment by Baspa** [comment]
Can you show me the config of your panel? What version are you on? @wuyidebh

**Reply by wuyidebh** [reply]
`<?php

namespace App\Providers\Filament;

use App\Filament\Auth\Login;
use App\Filament\Auth\Register;
use App\Filament\Auth\RequestPasswordReset;
use App\Filament\Resources\ProjectResource;
use CharrafiMed\GlobalSearchModal\GlobalSearchModalPlugin;
use Filament\FontProviders\LocalFontProvider;
use Filament\Http\Middleware\Authenticate;
use Filament\Http\Middleware\AuthenticateSession;
use Filament\Http\Middleware\DisableBladeIconComponents;
use Filament\Http\Middleware\DispatchServingFilamentEvent;
use Filament\Pages;
use Filament\Panel;
use Filament\PanelProvider;
use Filament\Support\Colors\Color;
use Filament\Widgets;
use Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse;
use Illuminate\Cookie\Middleware\EncryptCookies;
use Illuminate\Foundation\Http\Middleware\VerifyCsrfToken;
use Illuminate\Routing\Middleware\SubstituteBindings;
use Illuminate\Session\Middleware\StartSession;
use Illuminate\View\Middleware\ShareErrorsFromSession;

class AdminPanelProvider extends PanelProvider
{
    public function panel(Panel $panel): Panel
    {
        return $panel
            ->default()
            ->id('admin')
            ->path('admin')
            ->login(Login::class)
            ->registration(Register::class)
            ->profile()
            ->emailVerification()
            ->passwordReset(RequestPasswordReset::class)
            ->favicon('images/favicon.svg')
            ->emailVerification()
            ->brandName('WallCalc')
            ->brandLogoHeight('5rem')
            ->colors([
                'primary' => Color::Amber,
            ])
            ->font(
                'Inter',
                url: asset('css/customfonts.css'),
                provider: LocalFontProvider::class,
            )
            ->discoverResources(in: app_path('Filament/Resources'), for: 'App\\Filament\\Resources')
            ->discoverPages(in: app_path('Filament/Pages'), for: 'App\\Filament\\Pages')
            ->pages([
                Pages\Dashboard::class,
            ])
            ->discoverWidgets(in: app_path('Filament/Widgets'), for: 'App\\Filament\\Widgets')
            ->widgets([
                Widgets\AccountWidget::class,
                Widgets\FilamentInfoWidget::class,
            ])
            ->middleware([
                EncryptCookies::class,
                AddQueuedCookiesToResponse::class,
                StartSession::class,
                AuthenticateSession::class,
                ShareErrorsFromSession::class,
                VerifyCsrfToken::class,
                SubstituteBindings::class,
                DisableBladeIconComponents::class,
                DispatchServingFilamentEvent::class,
            ])
            ->authMiddleware([
                Authenticate::class,
            ])
            ->globalSearchKeyBindings(['command+k', 'ctrl+k']) 
            ->plugins([
                GlobalSearchModalPlugin::make()
                    ->closeByEscaping(enabled: true)
                    ->closeByClickingAway(enabled: true)
                    ->closeButton(enabled: false)
                    ->SwappableOnMobile(enabled: false)
                    ->highlightQueryStyles([
                        'background-color' => 'yellow',
                        'font-weight' => 'bold',
                    ])
                    ->searchItemTree(false)

            ]);
    }
}`

**Comment by wuyidebh** [comment]


**Reply by Baspa** [reply]
??

**Comment by wuyidebh** [comment]


**Reply by Baspa** [reply]
Please format your code.. @wuyidebh

**Comment by ca7vin** [comment]
I have the same problem here, displaying a full width form in place of the modal

Here is the config of my AdminPanel :

```
<?php

namespace App\Providers\Filament;

use App\Filament\Pages\Dashboard;
use App\Filament\Widgets\UserAccount;
use Filament\Http\Middleware\Authenticate;
use Filament\Http\Middleware\AuthenticateSession;
use Filament\Http\Middleware\DisableBladeIconComponents;
use Filament\Http\Middleware\DispatchServingFilamentEvent;
use Filament\Pages;
use Filament\Panel;
use Filament\PanelProvider;
use Filament\Support\Colors\Color;
use Filament\Widgets;
use Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse;
use Illuminate\Cookie\Middleware\EncryptCookies;
use Illuminate\Foundation\Http\Middleware\VerifyCsrfToken;
use Illuminate\Routing\Middleware\SubstituteBindings;
use Illuminate\Session\Middleware\StartSession;
use Illuminate\View\Middleware\ShareErrorsFromSession;
use Vormkracht10\TwoFactorAuth\TwoFactorAuthPlugin;

class AdminPanelProvider extends PanelProvider
{

    public function panel(Panel $panel): Panel
    {
        return $panel
            ->default()
            ->id('admin')
            ->path('admin')
            ->login()
            ->emailVerification()
            ->colors([
                'primary' => Color::Amber,
            ])
            ->discoverResources(in: app_path('Filament/Resources'), for: 'App\\Filament\\Resources')
            ->discoverPages(in: app_path('Filament/Pages'), for: 'App\\Filament\\Pages')
            ->pages([
                Dashboard::class,
            ])
            ->discoverWidgets(in: app_path('Filament/Widgets'), for: 'App\\Filament\\Widgets')
            ->widgets([
                Widgets\AccountWidget::class,
                // UserAccount::class,
            ])
            ->middleware([
                EncryptCookies::class,
                AddQueuedCookiesToResponse::class,
                StartSession::class,
                AuthenticateSession::class,
                ShareErrorsFromSession::class,
                VerifyCsrfToken::class,
                SubstituteBindings::class,
                DisableBladeIconComponents::class,
                DispatchServingFilamentEvent::class,
            ])
            ->authMiddleware([
                Authenticate::class,
            ])
            ->plugins([
                // TwoFactorAuthPlugin::make()->forced(),
                TwoFactorAuthPlugin::make(),
            ])
        ;
    }
}
```

**Comment by Baspa** [comment]
Can any of you provide me a reproduction repository @wuyidebh / @ca7vin

**Reply by ca7vin** [reply]
It's a private repository, i justed added you to the collaborators
[https://github.com/ca7vin/valora-app/](url) (branch = dev)

**Reply by ca7vin** [reply]
@Baspa Any news ? :)

**Reply by Baspa** [reply]
Didnt have time to take a look at it yet. Maybe next week @ca7vin

**Reply by ca7vin** [reply]
No worries, thank you very much !

PS : I also posted a Q&A about the mail verification for 2FA activation which is not sending there 
[discussion#93](https://github.com/vormkracht10/filament-2fa/discussions/93) and i didn't find any solution (if you have the time to check it, would be really nice)

Have a nice day !

**Comment by markvaneijk** [comment]
> Can any of you provide me a reproduction repository @wuyidebh / @ca7vin 

I think this happens particularly when the top navigation is configured in Filament.
